subsys/nrf_security/src/core/lite/psa_core_lite.c

```diff
@@ -0,0 +1,351 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "psa_core_lite.h"
+#include <psa/crypto.h>
+#include <psa_crypto_driver_wrappers.h>
+
+#if (defined(PSA_WANT_ALG_ECDSA) || defined(PSA_WANT_ALG_DETERMINISTIC_ECDSA)) && \
+	defined(PSA_WANT_ECC_SECP_R1_256)
+	const size_t pub_key_max_size = 65;
```

Comment:
Maybe better use the macros from the core PSA_EXPORT_KEY_OUTPUT_SIZE?

Reply:
Trying to avoid automatic configurations that may increase this buffer due to stray configurations. PSA core lite only supports Ed25519 (32 bytes key) and ECDSA using secp256r1 (uncompressed keys, 65 bytes, using SHA-256). If you haven't enabled any of these explicitly you are getting a failure at line 18.